components/filter-components/MapFilters.vue

```diff
@@ -40,15 +40,13 @@
 
               <!-- Display number of active witches. -->
               <div class="ml-3 flex mt-3 items-center pb-1">
-                <p class="mr-2 text-lg witchy-text">Showing</p>
-                <div
-                  class="h-6 px-1 flex items-center justify-center mr-2 border-2 rounded-md text-white font-medium bg-slate-500 border-slate-700"
+                <p class="mr-2 text-lg witchy-text mb-0">Showing</p>
+                <span
```

Comment:
Looks good Ruby. Only thing I noticed is that originally, we used a bold font for the numbers (and gender styling) but don't appear to any more. Is it ok to make it look like it did prior? Thank you

Reply:
font-medium doesnt seem to be working so have applied font-semibold, but may be too bold?

Reply:
think it looks ok so happy to merge in. Cheers Ruby